doc/changes/changes_0.1.0.md

```diff
@@ -1,4 +1,6 @@
-# Artifact Reference Checker Maven Plugin 0.1.0, released 2020-XX-XX
+# Artifact Reference Checker Maven Plugin 0.1.0, released 2020-09-04
+
+Code name: Initial implementation 
```

Comment:
I think `Code name` should be the issue or pull request number?

Reply:
no, It's a new part of the template tat determines the headline of the github release: https://raw.githubusercontent.com/exasol/release-robot/feature/16_improve_validation_and_release_process/doc/user_guide/changes_file_template.md

Reply:
Okay, I see. 
But it is still unclear for me if it is for users.